packages/nimble-components/src/table-column/text/template.ts

```diff
@@ -7,6 +7,9 @@ TableCellState<TableColumnTextCellRecord, TableColumnTextColumnConfig>
 >`
     <span
         class="${x => (typeof x.cellRecord.value === 'string' ? '' : 'placeholder')}"
+        title="${x => (typeof x.cellRecord.value === 'string'
+        ? x.cellRecord.value
```

Comment:
Maybe it's just me, but it seems like lines 11 & 12 should be indented one more level.

Reply:
I've changed the implementation since you looked, but it still has weird indentation. It's what prettier wants for some reason, and I don't know if it's worth adding a suppression for.